packages/block-editor/src/components/block-compare/style.scss

```diff
@@ -23,6 +24,7 @@
 		width: 50%;
 		padding: 0 $grid-unit-20 0 0;
 		min-width: 200px;
+		max-width: 600px;
```

Comment:
The value seems slightly arbitrary, maybe we can use `$break-small` directly, which amounts to the same?

Reply:
It's quite arbitrary 😄 . I've gone ahead with your suggestion.